sdk/servicebus/service-bus/src/models.ts

```diff
@@ -50,15 +50,40 @@ export interface CreateBatchOptions extends OperationOptions {
    * The upper limit for the size of batch. The `tryAdd` function will return `false` after this limit is reached.
    */
   maxSizeInBytes?: number;
+}
+
+/**
+ * The set of options to configure the behavior of the underlying operations.
+ *
+ * @export
+ * @interface OptionsBag
+ */
+export interface OptionsBag {
```

Comment:
Hm..premium real-estate of "RetryOptions" was already taken. 

Is this just "BasicOptions" or something of that nature?

Reply:
Yeah, `OptionsBag` is just a placeholder due to the lack of better names.
Let me know if you have any suggestions.

Reply:
Changed to BaseClientOptions.